docs/index.html

```diff
@@ -631,6 +631,38 @@ <h2 id="setreturnfocus-heading">setReturnFocus option</h2>
       </p>
     </div>
 
+    <div id="demo-setreturnfocus-function">
+      <h2 id="setreturnfocus-function-heading">setReturnFocus as function</h2>
+      <p>
+        As the function is executed in the process of deactivation, it allows you to dynamically specify which element should be focused after deactivation.
+      </p>
+      <p>
+        <button id="activate-setreturnfocus-function" aria-describedby="setreturnfocus-function-heading">
+          activate trap
+        </button>
+        <p id="deactivate-setreturnfocus-function">
+          <button id="focus-this">
+            deactivate and focus this
+          </button>
+          <button id="focus-initial">
+            deactivate and focus 'activate trap'
+          </button>
+          <button id="no-focus">
+            deactivate and no focus
```

Comment:
Could we label it "deactivate and no change in focus", since it'll just leave whatever's focused at the time of deactivation focused? Having just "no focus" makes me think it'll remove focus too, which it won't.